applications/crowdloan_frontend_template.md

```diff
@@ -0,0 +1,189 @@
+- **Total Costs:** 30,000 USD
+
+### Milestone 1
+
+- **Estimated duration:** 4-5 weeks
+- **FTE:**  2 (as above)
+- **Costs:** 30,000 USD
+
+| Number | Deliverable | Specification                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+| -----: | ----------- |--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 0a. | License | MIT or TBD                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| 0b. | Documentation | For both approaches (Webflow & Jamstack) we provide how-to guides. Jamstack will be documented in the Github repo.                                                                                                                                                                                                                                                                                                                                           |
+| 0c. | Article | We will publish an article presenting the templates and how to use them.                                                                                                                                                                                                                                                                                                                                                                                     |
+| 1a. | Benchmarking | Design research of existing Crowdloan pages and other one pages designs, like Github project pages templates. <br>This will allow us to synthesize viable options of the page designs                                                                                                                                                                                                                                                                        |
```

Comment:
Due to the overall scope of the project there is no document created, since that work informs the next step, which implicitly creates the documents. We can share the raw data in form of Miro boards with screenshots, Google Sheets or similar.
If every step is required to have a deliverable then I would rather rewrite the whole approach, to not have unnecessary overhead